docs/source/initialsetup/configuringmame.rst

```diff
@@ -0,0 +1,78 @@
+
+Once you have MAME installed, the next step is to configure it. There are
+several ways to do this, and each will be covered in turn.
+
+If you are on Windows, the MAME executable will be called ``mame64.exe`` or
+``mame.exe`` depending on if you have a 64-bit or 32-bit version of MAME.
+
+If you are on Linux or MacOS, the MAME executable will be called ``mame64`` or
+``mame`` depending on if you have a 64-bit or 32-bit version of MAME.
+
+
+Initial Setup: Creating mame.ini From Command Line on Windows
+-------------------------------------------------------------
+
+First, you will need to *cd* to the directory where you installed MAME into.
+If, for instance, you have MAME installed in ``c:\users\public\mame`` you will
+need to type ``cd c:\users\public\mame`` into the command prompt.
+
+Then you have MAME create the config file by typing ``mame64 -createconfig``
+or ``mame -createconfig`` depending on if you are using the 64-bit version of
+MAME or the 32-bit version.
+
+MAME will then create the ``mame.ini`` file in the MAME installation folder.
+This file contains the default configuration settings for a new MAME
+installation.
+
+
+Initial Setup: Creating mame.ini From Command Line on Linux or MacOS
+--------------------------------------------------------------------
+
+The steps for Linux and MacOS are similar to those of Windows. If you
+installed MAME using the package manager that came from a Linux distro, you will
+type ``mame64 -createconfig`` or ``mame -createconfig`` into your terminal
+of choice, depending on if you have a 64-bit version of MAME or a 32-bit
+one.
+
+If you have compiled from source or downloaded a binary package of MAME,
+you will ``cd`` into the directory you put the MAME files into.
+
+For instance, ``cd /home/myusername/mame``
+
+Then you will type ``./mame64 -createconfig`` or ``./mame -createconfig`` into
+your terminal of choice, depending on if you have a 64-bit version of MAME or a
+32-bit one.
+
+You can then need to edit the ``mame.ini`` file in your favorite text editor,
+e.g. *notepad* on Windows or *vi* on Linux/MacOS, or you can change settings
+from inside of MAME.
+
+
+Initial Setup: Graphical Setup
+------------------------------
+
+This is the easiest way to get started. Start MAME by opening the MAME icon
+in the location where you installed it. This will be ``mame64.exe`` or
+``mame.exe`` on Windows, ``mame64`` or ``mame`` on Linux and MacOS.
+
+Once MAME has started, you can either use your mouse to click on the
+**Configure Options** menu selection at the bottom center of your screen,
+or you can switch panes to the bottom one (default key is Tab), then press
+the menu accept button (default key is Enter) to go into the Configuration menu.
+
+Choose **Save Configuration** to create the ``mame.ini`` file with default
+settings. From here, you can either continue to configure things from the
+graphical user interface or edit the ``mame.ini`` file in your favorite
+text editor.
```

Comment:
Should probably mention where the mame.ini gets saved by default when you do this.  It goes in ~/Application Support/something-or-other on macOS, I’d have to check what it does on Linux.

Reply:
@rb6502 macOS location information, please?

Reply:
You can see what it will do here: https://github.com/mamedev/mame/blob/mame0226/src/osd/sdl/sdlmain.cpp#L54
* Save to working directory on Windows
* On macOS, save to ~/Application Support/mame
* On Linux, save to ~/.mame